scripts/utils.sh

```diff
@@ -187,13 +187,27 @@ function configure_none_platform_iptables_rules() {
 
 function get_pythonpath() {
   root_dir=$1
-  project_path=${root_dir}/src
+  project_path="${root_dir}/src"
   PYTHONPATH="${PYTHONPATH:-$project_path}"
 
-  if [[ ":${PYTHONPATH}:" != *":${project_path}:"* ]] && [[ ":${PYTHONPATH}:" != *":${project_path}/:"* ]];then
-    echo "${PYTHONPATH:+"$PYTHONPATH:"}$project_path" ;
-  else echo "${PYTHONPATH}";
+  update_path_var "${PYTHONPATH}" "$project_path"
+}
+
+function get_path() {
+  path="/usr/local/bin"
+  PATH="${PATH:-path}"
+  update_path_var "${PATH}" "$path"
+}
+
+function update_path_var() {
+  var_value=$1
+  path=$2
+
+  if [[ ":${var_value}:" != *":${path}:"* ]] && [[ ":${var_value}:" != *":${path}/:"* ]];then
+    echo "${var_value:+"$var_value:"}$path" ;
+  else echo "${var_value}";
```

Comment:
I don't have the context for this change but it seems like a lot to have here instead of:

`export PATH=$PATH:/usr/local/bin`

Are the changes to the PYTHONPATH variable intended?

Reply:
We often call make multiple times and if we export PATH as you suggested it Will append endlessly. We need to check if exists first.
We already doing that for PYTHONPAT, I just created a generic function for both of those vars

Reply:
I'm with @lranjbar with this one
Check above suggestion, it can save a lot of code (I tried running it multiple times, as expected it's not appending endlessly)